shims/common/src/main/java/org/apache/hadoop/hive/shims/Utils.java

```diff
@@ -172,6 +173,11 @@ public static boolean checkFileSystemXAttrSupport(FileSystem fs) throws IOExcept
       LOG.warn("XAttr won't be preserved since it is not supported for file system: " + fs.getUri());
       return false;
     }
+    catch (AccessControlException e) {
+      LOG.warn("Checking for XAttrSupport at {} Location", RAW_RESERVED_VIRTUAL_PATH);
+      fs.getXAttrs(new Path(RAW_RESERVED_VIRTUAL_PATH));
```

Comment:
How do you know you are gonna shoot a distcp on reserved path? Say you have Isilon system, but you aren't using the reserved path. In that case this will return true, but when you shoot an actual distcp, you will fail because you were using the normal paths